### PR TITLE
A little bit of pom cleanup.

### PR DESCRIPTION
**Pull request by jeffret-b**
Primarily about using https.

### CODE REVIEW

File: jrebel/pom.xml [hunk 1]
```diff
@@ -35,19 +35,4 @@
     </dependency>
   </dependencies>
 
-  <!--
```
Comment (jeffret-b) [inline jrebel/pom.xml]: This one isn't being used (commented out). The URL doesn't exist. And is http. Seems best to just remove this section.

File: pom.xml [hunk 2]
```diff
@@ -67,14 +67,14 @@
   <repositories>
     <repository>
       <id>repo.jenkins-ci.org</id>
-      <url>http://repo.jenkins-ci.org/public/</url>
+      <url>https://repo.jenkins-ci.org/public/</url>
```
Comment (jeffret-b) [inline pom.xml]: These don't work at either http or https but it's better convert them to https anyway.

**Comment by jeffret-b** [comment]
@daniel-beck @Wadeck @jvz

File: pom.xml (hunk 2 above)
Comment (Wadeck) [reply pom.xml]: WDYM? they are used by the publisher plugin, not meant to be accessible directly in the browser.

Comment (daniel-beck) [reply pom.xml]: Browsing at the virtual repo base URL is prohibited for performance reasons, e.g. https://repo.jenkins-ci.org/public/org/ works.

Comment (jeffret-b) [reply pom.xml]: Makes sense. Thanks.

File: pom.xml [hunk 3]
```diff
@@ -24,13 +24,13 @@
...
 
-  <url>http://stapler.kohsuke.org/</url>
+  <url>https://stapler.kohsuke.org/</url>
 
   <licenses>
     <license>
       <name>2-clause BSD license</name>
       <distribution>repo</distribution>
-      <url>http://opensource.org/licenses/bsd-license.php</url>
+      <url>https://opensource.org/licenses/bsd-license</url>
```
Comment (jglick) [inline pom.xml]:
```suggestion
      <url>https://opensource.org/licenses/BSD-2-Clause</url>
```

seems to be canonical, or is at least what is linked to from https://opensource.org/licenses.

Comment (jeffret-b) [reply pom.xml]: I've seen the first one a number of places, but the second should be goot.

**Pull request merged**